pfio/ClientManager.F90

```diff
@@ -5,7 +5,7 @@ module pFIO_ClientManagerMod
 
    use MAPL_ExceptionHandling
    use MAPL_SortMod
-   use pFIO_KeywordEnforcerMod
+   use mapl_KeywordEnforcerMod
```

Comment:
Random comment: Should this be `MAPL_KeywordEnforcerMod`? We seem to use the `MAPL` in most use statements.

Reply:
yes - how did I miss it?   Will never know.  Unfortunately I compiled before I deleted the old module, but I thought after I had removed it from CMake.

Reply:
I'm adopting a policy of making the namespace prefix be lower case.   This generally improves legibility IMO, as the prefix contains the least important information.   This is esp. true when scanning a long list where the prefix dominates the quick scan.   OTOH, I did not feel like pushing a change through all the files to enforce this.  Just doing it as I go.